### PR TITLE
Adding initial FAQ page.

### PR DESCRIPTION
**Pull request by ScottDowne**

### CODE REVIEW

**Comment by ScottDowne** [comment]
@TheoChevalier Here is the initial fa page. The strings are pretty much just like they are in the doc. We can tweak things that don't work well now that we have something usable.

I didn't do much with design, kept it pretty plain for now.

@alanmoo I'll probably lean on you for some review.

File: locales/en-US/messages.properties [hunk 1]
```diff
@@ -91,3 +91,52 @@ i_donated_to_thunderbird=I donated to @mozthunderbird today to #freetheinbox. Jo
...
+faq_item_2_paragraph_b=In certain instances, when you donate to Mozilla via a third party like PayPal or Stripe, those organizations also collect user data.
+faq_item_3_header=Why do you need my address in order to process a donation?
+faq_item_3_paragraph=We understand that your privacy is very important. We ask for a minimum amount of information required to process credit card donations, including billing addresses. This allows our payment processor to verify your identity, process your payment, and prevent fraudulent charges to your credit card. We keep your information private -- if you have questions, please refer to our <a href="https://www.mozilla.org/en-US/privacy/principles/">Privacy Policy</a>. If you would rather not fill in your information on our online donation form, you can <a href="https://wiki.mozilla.org/Ways_to_Give#Check_.28via_postal_service.29">mail a check</a> or donate directly <a href="https://sendto.mozilla.org/page/contribute/Give-Now">via Paypal</a>.
+faq_item_4_header=Where do I send a check/cheque?
+faq_item_4_paragraph_a=Please send your check to:
+faq_item_4_paragraph_b=Mozilla Foundation
```
Comment (TheoChevalier) [inline locales/en-US/messages.properties]: We should actually hardcode that and the rest of the address, we don’t want people to change anything to this US address

File: locales/en-US/messages.properties [hunk 2]
```diff
@@ -91,3 +91,52 @@ i_donated_to_thunderbird=I donated to @mozthunderbird today to #freetheinbox. Jo
...
+faq_item_2_paragraph_b=In certain instances, when you donate to Mozilla via a third party like PayPal or Stripe, those organizations also collect user data.
+faq_item_3_header=Why do you need my address in order to process a donation?
+faq_item_3_paragraph=We understand that your privacy is very important. We ask for a minimum amount of information required to process credit card donations, including billing addresses. This allows our payment processor to verify your identity, process your payment, and prevent fraudulent charges to your credit card. We keep your information private -- if you have questions, please refer to our <a href="https://www.mozilla.org/en-US/privacy/principles/">Privacy Policy</a>. If you would rather not fill in your information on our online donation form, you can <a href="https://wiki.mozilla.org/Ways_to_Give#Check_.28via_postal_service.29">mail a check</a> or donate directly <a href="https://sendto.mozilla.org/page/contribute/Give-Now">via Paypal</a>.
```
Comment (TheoChevalier) [inline locales/en-US/messages.properties]: https://www.mozilla.org/privacy/principles/ (without en-US)
Last link should be https://donate.mozilla.org/paypal-donate

File: locales/en-US/messages.properties [hunk 3]
```diff
@@ -91,3 +91,52 @@ i_donated_to_thunderbird=I donated to @mozthunderbird today to #freetheinbox. Jo
...
+faq_item_8_paragraph_a=Firefox is maintained by the Mozilla Corporation, a wholly-owned subsidiary of the Mozilla Foundation. While Firefox does produce revenue -- chiefly through search partnerships -- this earned income is largely reinvested back into the Corporation. The Mozilla Foundation's education and advocacy efforts, which span several continents and reach millions of people, are supported by philanthropic donations.
+faq_item_8_paragraph_b=Read more about the distinction <a href="http://mzl.la/1wSPI66">here</a>.
+faq_item_9_header=Who donates to Mozilla?
+faq_item_9_paragraph=Last year, more than 400,000 individuals gave to Mozilla to protect and improve the health of the internet. The Mozilla Foundation also receives support in the form of grants from other major foundations, like Knight, MacArthur and Ford.
+faq_item_10_header=Does my donation give me access to tech support?
+faq_item_10_paragraph=Anyone can access tech support for Mozilla’s products by visiting the <a href="https://support.mozilla.org/en-US/questions">support forum</a>. Mozilla does not provide tech support or enhanced tech support in exchange for donation.
```
Comment (TheoChevalier) [inline locales/en-US/messages.properties]: https://support.mozilla.org/questions

File: locales/en-US/messages.properties [hunk 4]
```diff
@@ -91,3 +91,52 @@ i_donated_to_thunderbird=I donated to @mozthunderbird today to #freetheinbox. Jo
 
 # Site title when shared on Facebook
 support_mozilla=Support Mozilla
+
+# faq page
+faq_header=FAQ
+faq_donate_link=CLICK HERE TO DONATE NOW
+faq_intro=Below are the most frequently asked questions about donating to the Mozilla Foundation. If your question and answer are not listed here, please contact us at <a href="mailto:donate@mozilla.org">donate@mozilla.org</a>.
+faq_item_1_header=What are the ways I can donate?
+faq_item_1_paragraph=<a href="https://donate.mozilla.org/">Visit this link to make a secure online donation right now via credit card or PayPal</a>. For additional ways to give -- like check, bitcoin, or bank transfer -- visit our <a href="https://wiki.mozilla.org/Ways_to_Give"Ways to Give page</a>.
```
Comment (TheoChevalier) [inline locales/en-US/messages.properties]: link tag not closed `<a href="https://wiki.mozilla.org/Ways_to_Give" _>_ Ways to Give page</a>`

File: locales/en-US/messages.properties [hunk 5]
```diff
@@ -91,3 +91,52 @@ i_donated_to_thunderbird=I donated to @mozthunderbird today to #freetheinbox. Jo
...
+faq_item_2_header=How does Mozilla ensure my personal information is protected?
+faq_item_2_paragraph_a=We care about the privacy of our donors and take our <a href="https://www.mozilla.org/en-US/privacy/principles/">Privacy Principles</a> seriously. Mozilla believes individuals should be in control of their data, and maintains multi-layered security controls and practices. To learn more, read the <a href="https://www.mozilla.org/en-US/privacy/">Mozilla Privacy Policy</a>.
```
Comment (TheoChevalier) [inline locales/en-US/messages.properties]: https://www.mozilla.org/privacy/principles/ and https://www.mozilla.org/privacy/

**Pull request closed without merging**